styles/trip-house-styles.css

```diff
@@ -36,8 +36,149 @@
 	border-top: 2px solid #f5bd41;
 }
 
-.offerings {
-	padding-top: 80px;
+/*header-block*/
+
+.header-section {
+	height: 100vh;
+	background-image: url("../images/header/background.svg");
+	background-size: cover;
+}
+
+.top-line {
+	padding-top: 50px;
+	padding-bottom: 200px;
+}
+
+.logo {
+	width: 205px;
+	height: 40px;
+	background-image: url("../images/header/sprite.png");
+}
+
+.navigation {
+	font-size: 24px;
+	color: #ffffff;
+	display: flex;
+	gap: 48px;
+}
+
+.header-links:hover {
+	text-decoration: underline #f5bd41 3px;
+	text-underline-offset: 12px;
+}
+
+.navigation-comm {
+	display: flex;
+	align-items: center;
+	gap: 24px;
+}
+
+.night {
+	width: 30px;
+	height: 30px;
+	background-image: url("../images/header/sprite.png");
+	background-position: -205px;
+}
+
+.account {
+	width: 40px;
+	height: 40px;
+	background-image: url("../images/header/sprite.png");
+	background-position: -235px;
+}
+
+.motto {
+	font-weight: 500;
+	font-size: 50px;
+	color: #ffffff;
+	padding-top: unset;
+	padding-bottom: 145px;
+}
+
+.search-description {
+	font-size: 18px;
+	color: #ffffff;
+	position: absolute;
+	top: -46px;
+	left: 24px;
+}
+
+.search-form {
+	width: 1232px;
```

Comment:
Use max-width here and width:100%. It is give you more flexibility on other screens